### DISCUSSION TITLE
ww3_multi with switch IC3 and Ice homogeneous field

### DISCUSSION THREAD

**Discussion by adegroodt** [NOAA-EMC/WW3#1232] [Q&A]
Dear WW3 community,

I am currently working with WW3 in a polar region with a multi grid. 
I would like to use the switch IC3 with the multi.nml (or multi.inp) and add the wave damping by ice as a homogeneous field. I have already tried it in the shel.nml and it works well, but I cannot find a way to implement the ice as a homogeneous field in the multi.nml.
In the shel I have:
- added the input of wind, current and ice concentration in the &INPUT_NML as "T" section + added ICE_PARAM3 as "H"
- activated the section of homogeneous field and added in &HOMOG_INPUT_NML:
HOMOG_INPUT(3)%NAME        = 'IC3'
HOMOG_INPUT(3)%VALUE1      = 917.0
 
In multi.nml It seems that the value in the homogeneous section is for a speed and direction only:
- is it actually possible to add Ice as homogenous field like in the shel.nml?
- if yes, how would it be possible to add the option of homogeneous field in the %MODEL_GRID_NML section, as there are only 3 options: "no", "native", "INPUT%NAME".
- what would be the default values that could be used in value(1) and value(2) of the homogenous section in multi.nml for IC3?

I would appreciate a lot your help,
Thank you!

**Comment by mickaelaccensi** [comment]
Hello,

Sorry to tell you that homogeneous input is not implemented in the multi grid. BTY, you can create a fake netcdf file with a constant value to provide as forcing field.

Best,

**Reply by adegroodt** [reply]
Hello Mickael,

Thank you for your answer, I will try that. 

Best regards,

**Reply by MadsBruunPoulsen** [reply]
Hi both,

I recently started a new discussion  (#1547 ) in this forum on a related question, but I did not receive an answer yet, unfortunately. @adegroodt , did you manage to run ww3_multi with IC3 switch using input netCDF files for e.g. sea ice density?

Merry christmas. 

Best regards,
Mads
